python/hsfs/core/online_store_sql_engine.py

```diff
@@ -209,14 +213,22 @@ def init_async_mysql_connection(self, options=None):
             "Prepared statements are not initialized. "
             "Please call `init_prepared_statement` method first."
         )
-        _logger.debug("Acquiring or starting event loop for async engine.")
-        loop = asyncio.get_event_loop()
-        _logger.debug(f"Setting up aiomysql connection, with options : {options}")
-        loop.run_until_complete(
-            self._set_aiomysql_connection(
-                len(self._prepared_statements[self.SINGLE_VECTOR_KEY]), options=options
-            )
+        _logger.debug(
+            "Fetching storage connector for sql connection to Online Feature Store."
+        )
+        self._online_connector = self._storage_connector_api.get_online_connector(
+            self._feature_store_id
         )
+        self._connection_options = options
+        self._hostname = util.get_host_name() if self._external else None
+
+        if util.is_runtime_notebook():
+            _logger.debug("Running in Jupyter notebook, applying nest_asyncio")
```

Comment:
Should this be a warning rather than debug?

Reply:
I think debug is fine as it is not really something user needs to do